src/main/scala/nl.knaw.dans.easy.solr4files/CommandLineOptions.scala

```diff
@@ -15,18 +15,23 @@
  */
 package nl.knaw.dans.easy.solr4files
 
-import org.rogach.scallop.{ScallopConf, ScallopOption, Subcommand, singleArgConverter}
+import org.rogach.scallop.{ ScallopConf, ScallopOption, Subcommand }
 
 class CommandLineOptions(args: Array[String], configuration: Configuration) extends ScallopConf(args) {
+  type UUID = String
+  type StoreName = String
+
   appendDefaultToDescription = true
   editBuilder(_.setHelpWidth(110))
   printedName = "easy-update-solr4files-index"
   private val SUBCOMMAND_SEPARATOR = "---\n"
```

Comment:
You forgot `version(configuration.version)` above this line.

Reply:
Did I forgot or did the archetype forgot?

Reply:
probably archetype forgot to add it. Maybe you could do a PR there?